make RIOT_CI_BUILD=1 CC_NOCOLOR=1 --no-print-directory -C ./tests/suit_manifest test
r
ssh -t rici@saclay.iot-lab.info 'socat - tcp:m3-11.saclay.iot-lab.info:20000' 
kex_exchange_identification: read: Connection reset by peer
Connection reset by 193.55.250.145 port 22
make[1]: *** [/home/runner/work/RIOT/RIOT/Makefile.include:880: cleanterm] Error 255
Traceback (most recent call last):
  File "/home/runner/work/RIOT/RIOT/tests/suit_manifest/tests/01-run.py", line 24, in <module>
    sys.exit(run(testfunc))
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/__init__.py", line 27, in run
    child = setup_child(timeout, env=os.environ,
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/spawn.py", line 95, in setup_child
    sync_child(child, env)
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/spawn.py", line 132, in sync_child
    utils.test_utils_interactive_sync(child,
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/utils.py", line 30, in test_utils_interactive_sync
    _test_utils_interactive_sync(child, retries, delay)
  File "/home/runner/work/RIOT/RIOT/dist/pythonlibs/testrunner/utils.py", line 17, in _test_utils_interactive_sync
    ret = child.expect_exact([ready_exp, pexpect.TIMEOUT], timeout=delay)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/pexpect/spawnbase.py", line 421, in expect_exact
    return exp.expect_loop(timeout)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/pexpect/expect.py", line 179, in expect_loop
    return self.eof(e)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/pexpect/expect.py", line 122, in eof
    raise exc
pexpect.exceptions.EOF: End Of File (EOF). Exception style platform.
<pexpect.pty_spawn.spawn object at 0x7f8cf3cb5100>
command: /usr/bin/make
args: [b'/usr/bin/make', b'cleanterm']
buffer (last 100 chars): ''
before (last 100 chars): '145 port 22\r\r\nmake[1]: *** [/home/runner/work/RIOT/RIOT/Makefile.include:880: cleanterm] Error 255\r\n'
after: <class 'pexpect.exceptions.EOF'>
match: None
match_index: None
exitstatus: 2
flag_eof: True
pid: 245924
child_fd: 5
closed: False
timeout: 10
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0.05
delayafterclose: 0.1
delayafterterminate: 0.1
searcher: searcher_string:
    0: 'READY'
    1: TIMEOUT
make: *** [/home/runner/work/RIOT/RIOT/makefiles/tests/tests.inc.mk:26: test] Error 1

Return value: 2
